pkg/network/node/ovscontroller.go

```diff
@@ -228,6 +228,48 @@ func (oc *ovsController) SetupOVS(clusterNetworkCIDR []string, serviceNetworkCID
 	return otx.Commit()
 }
 
+type podNetworkInfo struct {
+	vethName string
+	ip       string
+}
+
+// GetPodNetworkInfo returns network interface information about all currently-attached pods.
+func (oc *ovsController) GetPodNetworkInfo() (map[string]podNetworkInfo, error) {
```

Comment:
This really returns a list of connected sandboxes - can you name the function accordingly?

Reply:
Not really... "sandboxes" is mostly a kubelet/runtime concept. The veth name and IP address apply to the pod as a whole, hence this is returning pod networking information. Yes, it's indexed by sandbox ID, but that's just because that's the ID that ovsController's caller uses to identify the pods; from ovsController's POV it could just as easily be called "pod ID".